config/index.js

```diff
@@ -12,7 +12,11 @@ const config = {
         },
         core: {
             url: 'http://oms-core-elixir',
-            port: 4000
+            port: 4000,
+            user: {
+                login: process.env.CORE_LOGIN || 'admin@aegee.org',
+                password: process.env.CORE_PASSWORD || 'admin1234'
```

Comment:
put `5ecr3t`, it's better

Reply:
I was using the default pw of admin@aegee.org for the dev setup, it's better to be convenient here, what do you think? I'd rather change it in 2 places at once or will leave it there the way it is

Reply:
i think using 5ecr3t anywhere as default pw will help us solve many headaches

Reply:
makes sense. I'd rather update it in core as well then, or pass it as the env variable.